app/settings/surveys/targeted-surveys/targeted-edit.controller.js

```diff
@@ -118,6 +120,7 @@ function (
         if (isStepComplete($scope.targetedSurvey.stepOne)) {
             $scope.activeStep = 2;
             $scope.stepOneWarning = false;
+            $scope.getCountryCodes();
```

Comment:
Do you think we should get the country codes with the page load, rather than when step one is complete? For slow connections, it may be better to have that running in the background while the user completes step 1. Thoughts?

Reply:
I was thinking about that, not sure, at the same time, this divides the pageload into smaller pieces. But maybe the user is more accepting of longer loading-time at the inital-pageload? I'll make a test and compare...

Reply:
Or we could run that request once the rest of the page loads but before you move to the next step? This way we allow the page to be interactive and usable, but still get the country codes before the user moves on to step 2. Also this isn't really that big of a deal, so you can tell me to shut up if you want.

Reply:
I moved it to page-init, works fine now so merging